index.html

```diff
@@ -0,0 +1,171 @@
+          <div class="container text-start text-md-center text-light">
+            <p>
+              The exhibition includes seven video works and installations of
+              apps that contemplate how we make music, interact via
+              social media.
+            </p>
+          </div>
+        </div>
+        <div class="section mx-auto my-2 d-flex flex-md-column justify-content-md-center align-items-center">
+          <img class="ps-1" src="assets/mobile/program-activities/workshop.png" alt="Workshop Icon" width="64" height="58">
+          <h3 class="activity-name work-exhi text-md-center">Workshop</h3>
+          <div class="container text-start text-md-center text-light">
+            <p>
+              Since its inception in 2002 CF has become one of the most anticipated
+              community events worldwide branching out as a pop-up
+              events.
+            </p>
+          </div>
+        </div>
+        <div class="section mx-auto my-2 d-flex flex-md-column justify-content-md-center align-items-center">
+          <img class="ps-1" src="assets/mobile/program-activities/ignite.png" alt="ignite icon" width="64" height="58">
+          <h3 class="activity-name ignite text-md-center">CF Ignite</h3>
+          <div class="container text-start text-md-center text-light">
+            <p>CF is an excellent place to connect with other like-minded
+              musicians and industry professionals, which could lead to new and
+              fruitful connections.
+            </p>
+          </div>
+        </div>
+        <div class="join-section text-center d-md-none mx-auto rounded-1">
+          <p class="join p-4">Join CF Edition 2023</p>
+        </div>
+      </div>
+      <div class="text-center py-5">
+        <a href="#" class="text-decoration-underline d-none text-light d-md-block">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+      <div class="performers-main-container">
+        <div class="performers-section py-3">
+        </div>
+      </div>
+  </main>
+  <footer>
+    <section id="sponsors" class="container-fluid d-none d-md-block">
+      <div class="d-flex flex-wrap gap-3 partners justify-content-center">
+        <div class="container container-md">
+          <h3 class="text-center text-light">Partners</h3>
+          <span class="orange-bar mx-auto"></span>
+        </div>
+        <div class="partners-logo">
+          <img class="mx-auto" src="assets/mobile/partners/microsoft.png" alt="microsoft logo" height="100" width="200">
+          <img class="mx-auto" src="./assets/mobile/partners/google.png" alt="google logo" width="200" height="85">
+          <img class="mx-auto" src="./assets/mobile/partners/pixar.png" alt="pixar logo" width="300" height="100">
+          <img class="mx-auto" src="./assets/mobile/partners/microverse.png" alt="microverse logo" width="400" height="99">
+        </div>
+      </div>
+    </section>
+  </footer>
+
```

Comment:
- [ ] As it is in the provided template, Kindly make sure you have the footer section visible on your home page.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203994309-87adebee-eb57-4825-a48d-753d7b8bf689.png)

behance design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203994420-c646ad7a-0616-40f0-afa9-7b1d269127d9.png)